unzipper.php

```diff
@@ -138,100 +145,111 @@ public static function extractGzipFile($archive, $destination) {
+            border-radius: 10px;
+            -webkit-box-shadow: rgba(255, 255, 255, 0.4) 0 0px 0, inset rgba(000, 000, 000, 0.7) 0 0px 0px;
+            -moz-box-shadow: rgba(255, 255, 255, 0.4) 0 0px 0, inset rgba(000, 000, 000, 0.7) 0 0px 0px;
+            box-shadow: rgba(255, 255, 255, 0.4) 0 0px 0, inset rgba(000, 000, 000, 0.7) 0 0px 0px;
+            padding: 8px;
+            margin-bottom: 20px;
+            width: 280px;
+        }
+        
+        .submit {
+            -moz-box-shadow: inset 0px 1px 0px 0px #bbdaf7;
+            -webkit-box-shadow: inset 0px 1px 0px 0px #bbdaf7;
+            box-shadow: inset 0px 1px 0px 0px #bbdaf7;
+            background: -webkit-gradient(linear, left top, left bottom, color-stop(0.05, #79bbff), color-stop(1, #378de5));
+            background: -moz-linear-gradient(center top, #79bbff 5%, #378de5 100%);
+            filter: progid: DXImageTransform.Microsoft.gradient(startColorstr='#79bbff', endColorstr='#378de5');
+            background-color: #79bbff;
+            -moz-border-radius: 4px;
+            -webkit-border-radius: 4px;
+            border-radius: 4px;
+            border: 1px solid #84bbf3;
+            display: inline-block;
+            color: #ffffff;
+            font-family: arial;
+            font-size: 15px;
+            font-weight: bold;
+            padding: 10px 24px;
+            text-decoration: none;
+            text-shadow: 1px 1px 0px #528ecc;
+        }
+        
+        .submit:hover {
+            background: -webkit-gradient(linear, left top, left bottom, color-stop(0.05, #378de5), color-stop(1, #79bbff));
+            background: -moz-linear-gradient(center top, #378de5 5%, #79bbff 100%);
+            filter: progid: DXImageTransform.Microsoft.gradient(startColorstr='#378de5', endColorstr='#79bbff');
+            background-color: #378de5;
+        }
+        
+        .submit:active {
+            position: relative;
+            top: 1px;
+        }
+        /* This imageless css button was generated by CSSButtonGenerator.com */
+    </style>
 </head>
 
 <body>
-<h1>Archive Unzipper</h1>
+    <h1>Archive Unzipper</h1>
 
-<p>Select .zip archive or .gz file you want to extract:</p>
+    <p>Select .zip archive or .gz file you want to extract:</p>
 
-<form action="" method="POST">
-  <fieldset>
+    <form action="" method="POST">
+        <fieldset>
 
-    <select name="zipfile" size="1" class="select">
+            <select name="zipfile" size="1" class="select">
       <?php foreach ($arc->zipfiles as $zip) {
         echo "<option>$zip</option>";
       }
       ?>
     </select>
-
-    <br/>
-
-    <input type="submit" name="submit" class="submit" value="Unzip Archive"/>
-
-  </fieldset>
-</form>
-<p class="status">
-  Status: <?php echo $arc::$status; ?>
-  <br/>
-  Processingtime: <?php echo $time; ?> ms
-</p>
+            <br />
+            <br/>
+            <input type="text" name="extpath" class="form-field" placeholder="/extraction_patch" />
```

Comment:
> you should not suggest an absolute path here, so instead of "/extraction_path" better "extraction_path"
path is still absolute which will break extraction I guess (not tested)